fv/test-connection/test-connection.go

```diff
@@ -126,7 +130,10 @@ func tryConnect(ipAddress, port, sourcePort, protocol, loopFile string) error {
 	testMessage := "hello," + uid
 
 	// The reuse library implements a version of net.Dialer that can reuse UDP/TCP ports, which we
-	// need in order to make connection retries work.
+	// need in order to make connection retries work. Since we specify the source port
+	// rather than use an ephemeral port, if the SO_REUSEADDR and SO_REUSEPORT options
+	// are not set, when we make another call to this program, the original port is in post-close
+	// wait state and bind fails.
```

Comment:
I don't know much about the reuse library or sockets but it looks like this comment might just apply to sctp, and if so this would fit better where we're doing the sctp dialing.

Reply:
The reuse library is for TCP/UDP, but the bit about the SO_REUSE* socket options applies to all connection types.

Reply:
I've updated this comment to hopefully make this more clear